### DISCUSSION TITLE
Log into a private Docker Registry

### DISCUSSION THREAD

**Discussion by smfelsher** [louislam/dockge#221] [Ask for Help]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I'm trying to launch a stack with multiple images. Two (2) of the images need to be downloaded from a GitLab Container Registry. To achieve this on the command line, I need to `docker login` first, then I can download the images from the registry.

Is there a way tp achieve this with `dockge`?

### 📝 Error Message(s) or Log

```
[+] Running 3/3
 ✘ db Error                                                                                         0.3s 
 ✘ db-backup Error                                                                                  0.3s 
 ✘ app Error                                                                                        0.3s 
Error response from daemon: Head "https://registry.gitlab.com/v2/edge-ovens/esa/esa-app/manifests/2.4.22": denied: access forbidden
```

### 🐻 Dockge Version

1.2.0

### 💻 Operating System and Arch

Manjaro Linux

### 🌐 Browser

Version 1.60.125 Chromium: 119.0.6045.199 (Official Build) (64-bit)

### 🐋 Docker Version

Docker 24.0.7

### 🟩 NodeJS Version

16.16.0

**Comment by louislam** [comment]
```yaml
    volumes:
      # If you want to use private registries, you need to share the auth file with Dockge:
       - /root/.docker/:/root/.docker
```

**Reply by smfelsher** [reply]
Sorry, I just looked right past that in the Docker Compose file. Thank you for responding.
